README.md

```diff
@@ -182,3 +182,63 @@ as both runs will use separate caches, but providing the shared key will lead to
   with:
     prefix-key: v1
 ```
+
+## Using the error message as output
+
+In case the semver check fails, this action will populate the `error_message` output.
+
+[An output can be used in other steps](https://docs.github.com/en/actions/using-jobs/defining-outputs-for-jobs), for example to comment the error message onto the pull request.
+
+```yml
+name: "Check PR"
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  check-semver:
+    name: Check semver
+    runs-on: ubuntu-latest
+    outputs:
+      error_message: ${{ steps.check_semver.outputs.error_message }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Run cargo-semver-checks
+        id: check_semver
+        uses: obi1kenobi/cargo-semver-checks-action@v2
+
+  comment-on-pr:
+    name: Comment on pull request
+    runs-on: ubuntu-latest
+    needs: check-semver
+    if: always()
+    permissions:
+      pull-requests: write
```

Comment:
Oh interesting! Does this work correctly even if the PR comes from a fork of the repo? 👀

I would have assumed this might give the job "PR write" permissions _on the fork_, but admittedly I find the GitHub Actions permissions model confusing and unpredictable even after working with it for years.

Reply:
Yeah I find it confusing as well. I need to find some time to set up a repo to test this - running out of time now.

Reply:
If you get a chance to test this, I'd love to make sure it works well before we put it in the docs.

Reply:
I never got a chance to dig too deep into this, but last time I looked, one possible answer seemed to be something like "create a GitHub App." That seemed a bit annoying to do, so I didn't do it at the time.

If you find or hear about any other possible workarounds, I'd love to know about them.

Reply:
Ah, I see. Well, I'm not sure then. Creating a GitHub app definitely sounds annoying :D